src/patternfly/components/Page/page.scss

```diff
@@ -224,12 +232,23 @@
 .pf-c-page__header-nav {
   grid-column: 1 / -1;
   grid-row: 2 / 3;
+  min-width: 0;
```

Comment:
Is the min-width just for FF?

Reply:
no, it breaks in chrome without it, too. the default is `min-width: min-content;` so by default, the header-nav element doesn't overflow and just grows to fit it's content. That causes the nav and masthead to grow too wide, resulting in this brokenness.

<img width="890" alt="Screen Shot 2019-05-17 at 9 31 53 AM" src="https://user-images.githubusercontent.com/35148959/57935081-9f97fe80-7886-11e9-9239-0955677cd783.png">

Reply:
ok great